instrumentation/action_mailer/lib/opentelemetry/instrumentation/action_mailer/instrumentation.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+
+# Copyright The OpenTelemetry Authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+module OpenTelemetry
+  module Instrumentation
+    module ActionMailer
+      # The Instrumentation class contains logic to detect and install the ActionMailer instrumentation
+      class Instrumentation < OpenTelemetry::Instrumentation::Base
+        MINIMUM_VERSION = Gem::Version.new('6.1.0')
+        install do |_config|
+          require_dependencies
+        end
+
+        present do
+          defined?(::ActionMailer)
+        end
+
+        compatible do
+          gem_version >= MINIMUM_VERSION
+        end
+
+        option :disallowed_notification_payload_keys, default: [],  validate: :array
```

Comment:
Do you mean like we change the logic of [span_subscriber](https://github.com/open-telemetry/opentelemetry-ruby-contrib/blob/opentelemetry-instrumentation-active_support/v0.5.1/instrumentation/active_support/lib/opentelemetry/instrumentation/active_support/span_subscriber.rb#L105) so it can reflect to all instrumentations that depend on it?

Reply:
yes. Again I do not think that needs to be done here.